### PR TITLE
release 1.2.2

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

File: src/main/resources/test.yml
Comment (sbouchet) [inline src/main/resources/test.yml:1]: is this file mandaotry for release ? maybe a orphan file from local Unit Testing ?

Comment (adietish) [reply src/main/resources/test.yml:1]: I removed it bcs it was erroneously added when testing something.

Comment (sbouchet) [reply src/main/resources/test.yml:1]: my bad. did not notice it was actually a removal of that file.

**Review by sbouchet — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
plugin 1.2.2.286 submitted to marketplace for approval

**Pull request merged**